cmd/zetae2e/local/solana.go

```diff
@@ -0,0 +1,60 @@
+package local
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fatih/color"
+
+	"github.com/zeta-chain/zetacore/e2e/config"
+	"github.com/zeta-chain/zetacore/e2e/e2etests"
+	"github.com/zeta-chain/zetacore/e2e/runner"
+)
+
+// solanaTestRoutine runs Solana related e2e tests
+func solanaTestRoutine(
+	conf config.Config,
+	deployerRunner *runner.E2ERunner,
+	verbose bool,
+	testNames ...string,
+) func() error {
+	return func() (err error) {
+		// initialize runner for solana test
+		solanaRunner, err := initTestRunner(
+			"solana",
+			conf,
+			deployerRunner,
+			conf.AdditionalAccounts.UserSolana,
+			runner.NewLogger(verbose, color.FgCyan, "solana"),
+		)
+		if err != nil {
+			return err
+		}
+
+		solanaRunner.Logger.Print("🏃 starting Solana tests")
+		startTime := time.Now()
+		solanaRunner.SetupSolanaAccount()
+
+		// run solana test
+		testsToRun, err := solanaRunner.GetE2ETestsToRunByName(
+			e2etests.AllE2ETests,
+			testNames...,
+		)
+		if err != nil {
+			return fmt.Errorf("solana tests failed: %v", err)
+		}
+
+		if err := solanaRunner.RunE2ETests(testsToRun); err != nil {
+			return fmt.Errorf("solana tests failed: %v", err)
+		}
+
+		// TODO: check Solana balance
+		// if err := solanaRunner.CheckSolanaTSSBalance(); err != nil {
+		// 	return err
+		// }
```

Comment:
Why commented out for now?
Need issue to track TODOs

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/eb814a91d89bee5be48174081af7338f76cb197a